proxywasm/log/log.go

```diff
@@ -0,0 +1,68 @@
+package log
```

Comment:
could you simply put in the `proxywasm` pkg rather than creating a new one?

Reply:
This would cause a circular dependency because:
- `proxywasm` depends on `internal`
- `internal` depends on the logging level for the `ProxyLog` function

I can move the code to the `types` package if you think this is a better fit.